docs/book/mlops-stacks/orchestrators/tekton.md

```diff
@@ -0,0 +1,139 @@
+* you're willing to deploy and maintain Tekton Pipelines
+on your cluster.
+
+## How to deploy it
+
+You'll first need to
+set up a Kubernetes cluster and deploy Tekton Pipelines:
+
+{% tabs %}
+{% tab title="AWS" %}
+
+* Have an existing
+  AWS [EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html)
+  set up.
+* Make sure you have the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) set up.
+* Download and [install](https://kubernetes.io/docs/tasks/tools/) `kubectl`
+  and [configure](https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-connection/)
+  it to talk to your EKS cluster using the following command:
+
+  ```powershell
+  aws eks --region REGION update-kubeconfig --name CLUSTER_NAME
+  ```
+* [Install](https://tekton.dev/docs/pipelines/install/)
+  Tekton Pipelines onto your cluster.
+  {% endtab %}
+
+{% tab title="GCP" %}
+
+* Have an existing
+  GCP [GKE cluster](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+  set up.
+* Make sure you have the [Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk) set up first.
+* Download and [install](https://kubernetes.io/docs/tasks/tools/) `kubectl`
+  and [configure](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
+  it to talk to your GKE cluster using the following command:
+
+  ```powershell
+  gcloud container clusters get-credentials CLUSTER_NAME
+  ```
+* [Install](https://tekton.dev/docs/pipelines/install/)
+  Tekton Pipelines onto your cluster.
+  {% endtab %}
+
+{% tab title="Azure" %}
+
+* Have an
+  existing [AKS cluster](https://azure.microsoft.com/en-in/services/kubernetes-service/#documentation)
+  set up.
+* Make sure you have the [`az` CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) set up first.
+* Download and [install](https://kubernetes.io/docs/tasks/tools/) `kubectl` and
+  it to talk to your AKS cluster using the following command:
+
+  ```powershell
+  az aks get-credentials --resource-group RESOURCE_GROUP --name CLUSTER_NAME
+  ```
+* [Install](https://tekton.dev/docs/pipelines/install/)
+  Tekton Pipelines onto your cluster.
+{% endtab %}
+{% endtabs %}
+
+{% hint style="info" %}
+If one or more of the deployments are not in the `Running` state, try increasing
+the number of nodes in your cluster.
+{% endhint %}
+
+{% hint style="warning" %}
+ZenML has only been tested with Tekton Pipelines >=0.38.3 and may not work with previous 
+versions.
+{% endhint %}
+
+## How to use it
+
+To use the Tekton orchestrator, we need:
+* The ZenML `tekton` integration installed. If you haven't done so, run 
+    ```shell
+    zenml integration install tekton -y
+    ```
+* [Docker](https://www.docker.com) installed and running.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) installed.
+
+When using the Tekton orchestrator with a remote cluster, you'll additionally need
```

Comment:
There is no `local` case at the moment, I think we can remove this sentence right?

Reply:
Hmm but the local case isn't mentioned here. Do you mean saying the word "additionally"?

Reply:
Yeah, I think we could just merge these two lists and remove the intermediate sentence, these are all equivalent requirements to run it.